common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -375,6 +394,20 @@ def editable_metadata_fields(self):
         else:
             editable_fields.pop('source')
 
+        licensable = False
+        course_id = None
+        course = None
+        if hasattr(settings, 'FEATURES') and settings.FEATURES.get('CREATIVE_COMMONS_LICENSING', False):
+            if hasattr(self.runtime, 'modulestore'):
+                course_id = self.runtime.course_id
+                if course_id:
+                    course = self.runtime.modulestore.get_course(course_id)
```

Comment:
The course cannot be found using this runtime.
When running a debugger, I get the following values:
```
self.runtime = CombinedSystem(None, <xmodule.modulestore.mongo.base.CachingDescriptorSystem object at 0x7f30de2c6fd0>)
self.runtime.modulestore = <xmodule.modulestore.mongo.draft.DraftModuleStore object at 0x7f30dde764d0>
course_id = CourseLocator(u'CCX', u'CC001', u'2015_T1', 'draft', None)
course = None
```